view/header/index.css

```diff
@@ -17,30 +17,31 @@
   margin-top: -2em;
 
   @media (min-width: 768px) {
-    width: 80%;
+    width: 100%;
   }
 }
 
 .header__more {
   display: inline-block;
-  font-size: 5vw;
-  font-weight: 300;
-  line-height: 1.5;
+  font-size: .5vw !important;
```

Comment:
Let's remove `!important`

Reply:
`.5vw` not readable at small devices

![image](https://user-images.githubusercontent.com/22644149/204136308-3a815d3b-5c6b-4bab-9dd3-0df3a94f550d.png)